web/modules/custom/jcc_custom/jcc_custom.module

```diff
@@ -991,8 +1004,33 @@ function jcc_custom_menu_local_tasks_alter(&$data, $route_name, RefinableCacheab
   if (isset($data['tabs'], $data['tabs'][0], $data['tabs'][0]['entity.paragraphs_library_item.collection'])) {
     unset($data['tabs'][0]['entity.paragraphs_library_item.collection']);
   }
+
+  //TCI-1268 hide the "Create new account" tab and its associated panel if the site is not newsroom nor superior
+  // and if the request path is in the following formats: '/user/register', '/user/login', '/user/password' and '/user'
+  $currentHost = Request::createFromGlobals()->getHost();
```

Comment:
not needed.  config in user.settings will hide this tab from anonymous users, which only take effect for sites using immutable.  newsroom and supremecourt are not part of immutable.